solutions/standard/main.tf

```diff
@@ -154,113 +166,96 @@ resource "ibm_is_subnet" "client_to_site_subnet_zone_2" {
   count           = var.vpn_subnet_cidr_zone_2 != null && length(var.existing_subnet_names) == 0 ? 1 : 0
   depends_on      = [ibm_is_vpc_address_prefix.client_to_site_address_prefixes_zone_2]
   name            = "${var.prefix}-client-to-site-subnet-2"
-  vpc             = var.vpc_id != null ? var.vpc_id : local.vpc_id
+  vpc             = local.existing_vpc_id
   ipv4_cidr_block = var.vpn_subnet_cidr_zone_2
   zone            = local.zone_2
   network_acl     = ibm_is_network_acl.client_to_site_vpn_acl[0].id
 }
 
-# workaround for https://github.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/issues/45
-resource "time_sleep" "wait_for_security_group" {
-  depends_on       = [module.client_to_site_sg.ibm_is_security_group]
-  create_duration  = "10s"
-  destroy_duration = "60s"
-}
-
 module "vpn" {
   source                        = "../.."
   depends_on                    = [time_sleep.wait_for_security_group]
-  server_cert_crn               = module.secrets_manager_private_certificate.secret_crn
-  vpn_gateway_name              = local.vpn_gateway_name
+  server_cert_crn               = local.server_cert_crn
+  vpn_gateway_name              = var.prefix != null ? "${var.prefix}-${var.name}" : var.name
   resource_group_id             = module.resource_group.resource_group_id
-  subnet_ids                    = length(var.existing_subnet_names) > 0 ? local.subnets : var.vpn_subnet_cidr_zone_2 != null ? [ibm_is_subnet.client_to_site_subnet_zone_1[0].id, ibm_is_subnet.client_to_site_subnet_zone_2[0].id] : [ibm_is_subnet.client_to_site_subnet_zone_1[0].id]
+  subnet_ids                    = local.subnet_ids
   create_policy                 = var.create_policy
   vpn_client_access_group_users = var.vpn_client_access_group_users
   access_group_name             = "${var.prefix}-${var.access_group_name}"
-  secrets_manager_id            = local.sm_guid
+  secrets_manager_id            = module.existing_sm_crn_parser.service_instance
   vpn_server_routes             = var.vpn_server_routes
 }
 
+# workaround for https://github.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/issues/45
```

Comment:
This issue seems to be closed.

Reply:
issue was closed, because workaround has been implemented and it is still needed, therefore I left it